### PR TITLE
Keep alive infinite loop

### PR DESCRIPTION
**Pull request by vishalkuo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -13,12 +13,14 @@ import(
 func initiate_get_request() {
```
Comment (tanatorn) [inline main.go]: nit: camel case?

File: main.go [hunk 2]
```diff
@@ -13,12 +13,14 @@ import(
...
   var resp *http.Response
```
Comment (tanatorn) [inline main.go]: we could declare these variables together - something like this:

``` go
var (
    resp   *htttp.Response,
    err    error
)
```

File: main.go [hunk 3]
```diff
@@ -13,12 +13,14 @@ import(
...
   var err error
-  resp, err = http.Get(os.Getenv("API_ENDPOINT"))
-  if err != nil {
-    panic(err)
+  for {
+    resp, err = http.Get(os.Getenv("API_ENDPOINT"))
+    if err != nil {
+      panic(err)
+    }
+    defer resp.Body.Close()
```
Comment (tanatorn) [inline main.go]: pretty sure if we defer this, we'll never actually close it because it runs in an infinite loop. Mem leak?

**Review by tanatorn — CHANGES_REQUESTED** [review]

**Comment by vishalkuo** [comment]
@tanatornn96 please review again when you get a chance

**Comment by tanatorn** [comment]
LGTM

**Review by tanatorn — APPROVED** [review]

**Pull request merged**